000-cbdb-sandbox/Dockerfile.RELEASE.centos7

```diff
@@ -47,17 +52,30 @@ RUN     cat /tmp/sysctl.conf.add >> /etc/sysctl.conf \
 RUN     yum install -y python3-devel python3-pip; yum clean all
 
 RUN     cd /tmp/ \
-        && unzip -d /tmp /tmp/cloudberrydb-1.5.2.zip \
-        && cd /tmp/cloudberrydb-1.5.2 \
-        && pip3 install -i https://mirrors.aliyun.com/pypi/simple -r python-dependencies.txt
+        && unzip -d /tmp /tmp/cloudberrydb-${CODEBASE_VERSION_VAR}.zip \
+        && cd /tmp/cloudberrydb-${CODEBASE_VERSION_VAR} \
+        && if [ -f python-dependencies.txt ]; then \
+              pip3 install -i https://mirrors.aliyun.com/pypi/simple -r python-dependencies.txt; \
```

Comment:
Can we delete the Aliyun mirrors here? I think it may become slower for the users to install in other regions. How about it?